README.md

```diff
@@ -77,31 +81,40 @@ yarn add <package-name>
 Whenever you install new libraries using yarn, make sure to update your
 `remappings.txt`.
 
-## Deploy and test using local fork
+### Deploy and test using local fork
+
+Deploy contracts to the local fork
 
 ```bash
 just deploy-local
 ```
 
-The result of deployment is `./artifacts/local/deploy-devnet.json` deployment config, which is required for integration testing
-
-Integration tests should pass either before a vote, or after
+Set up environment for the local fork
+Further test commands require the following environment variables to be set:
 
 ```bash
-just deploy-local
 export RPC_URL=http://127.0.0.1:8545
 export DEPLOY_CONFIG=./artifacts/local/deploy-devnet.json
+```
+
+The result of deployment is `./artifacts/local/deploy-devnet.json` deployment config, which is required for integration testing
+
+Verify deploy by running post-deployment tests.
+Note that these are meant to be run right after deployment, so they might fail on non-empty contracts
```

Comment:
yes it's expected, we should verify deployment together with these tests, after that it doesn't makes any sense anymore

Reply:
Then perhaps it would make sense to write explicitly that this is one-time test that must be run right after the deployment and there is no guarantee that they will pass after a while

Reply:
I mean `non-empty contracts` misleads me a bit